cmd/kubepkg/templates/latest/deb/kubeadm/debian/changelog

```diff
@@ -1,6 +1,5 @@
 kubeadm ({{ .Version }}-{{ .Revision }}) {{ .Channel }}; urgency=medium
 
-  * https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md
+  * https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/README.md
```

Comment:
Prefer the vanity URL (`https://git.k8s.io/kubernetes/CHANGELOG/README.md`) in user-facing documentation.
Same comment throughout the PR.

Reply:
Changed as suggested.